pages/sign_in.js

```diff
@@ -170,121 +172,96 @@ class SignIn extends React.Component {
+                  value={this.state.clientPassword}
+                  onChange = {(e) => this.handlePassword(e)}
+                  onBlur= {() => this.onBlurHandlePassword()}>
+                </Input>
+                </Form.Item>
+                :
+                <Form.Item>
+                <Input
+                size='medium'
+                type='password'
+                placeholder='Password'
+                prefix={<LockOutlined style={{color: '#267FFF'}}/>}
+                value={this.state.clientPassword}
+                onBlur= {() => this.onBlurHandlePassword()}
+                onChange = {(e) => this.handlePassword(e)}>
+                </Input>
+                </Form.Item>
               }
-            </form>
-          </a>
-        </div>
-        <style jsx>{`
-          .hero {
-          width: 100%;
-          color: #333;
-          }
-          .title {
-          margin: 0;
-          width: 100%;
-          padding-top: 80px;
-          line-height: 1.15;
-          font-size: 48px;
-          }
-          .title,
-          .description {
-          text-align: center;
-          }
-          .row {
-          max-width: 880px;
-          margin: 80px auto 40px;
-          display: flex;
-          flex-direction: row;
-          justify-content: space-around;
-          }
-          .card {
-          padding: 18px 18px 24px;
-          width: 220px;
-          text-align: left;
-          text-decoration: none;
-          color: #434343;
-          border: 1px solid #9b9b9b;
-          }
-          .card:hover {
-          border-color: #067df7;
-          }
-          .card h3 {
-          margin: 0;
-          color: #067df7;
-          font-size: 18px;
-          }
-          .card p {
-          margin: 0;
-          padding: 12px 0 0;
-          font-size: 13px;
-          color: #333;
-          }
-        `}</style>
-      </div>
+              <Form.Item>
+                <Button
+                  type="primary" block
+                  shape="round"
+                  onClick={(e) => this.handleClick(e)}>
+                  Login
+                </Button>
+              </Form.Item>
+              <Form.Item>
+                <p> or sign in with </p>
+                <Button
+                  type="primary" block
+                  shape="round"
+                  style={{background: '#F2F6FF', border: 'none', color: '#4E6991'}}
+                  icon={<GoogleCircleFilled style={{color: '#267FFF'}}/>}
+                  onClick={this.handleSignInWithGoogle}>
+                  Google
+                </Button>
+              </Form.Item>
+              <span> Don't have an account yet? </span>
+              <Link href="/sign_up">
+                <a>Sign up</a>
+              </Link>
+            </Form>
+          </Card>
+        </Col>
+      </Row>
+      </React.Fragment>
     )
   }
 }
 
 export default withRedux(SignIn)
-
```

Comment:
why remove the extra line at the end in this change request?

Reply:
Got it. Fixing it now